maintenance/aws-janitor/main.go

```diff
@@ -48,48 +50,58 @@ func main() {
 	// to delete.
 	sess := session.Must(session.NewSessionWithOptions(session.Options{Config: aws.Config{MaxRetries: aws.Int(100)}}))
 
+	if *cleanAll {
+		if err := resources.CleanAll(sess); err != nil {
```

Comment:
Do we want to ignore the user passed `-region` flag here? Or should that be passed along to `CleanAll`?

Reply:
Hi @detiber,

Good catch, thank you! I've fixed the issue in a force push.